cli/configApp.js

```diff
@@ -68,10 +68,18 @@ const API_TARGETS = {
 
 const SECURE_URL = ENV_CONFIG.secure;
 
+const identity = (v) => v;
+
 function main({ api = 'dev' }) {
-    const [mainApi, extendedApi] = api.split('+');
-    const apiUrl = API_TARGETS[extendedApi || mainApi] || API_TARGETS.prod;
-    const secureUrl = SECURE_URL[api] || SECURE_URL.prod;
+    const apis = api.split('+');
+    const apiUrl = ['prod', ...apis]
+        .map((key) => API_TARGETS[key])
+        .reverse()
+        .find(identity);
```

Comment:
I don't understand the process here ? What do you want and how does it work ?

Reply:
I map the api names to the configured url, and find the last value that is not empty.

The `['prod',` is to default the value to prod, in case the other values are empty, just as it was being done before.

Reply:
Personally, I'd rather merge all the objects under each of the provided api names, and then read config directly from that. This would be a larger change though.

Reply:
Hooo ok you take the first item of the list. Can you remove the find and use [apiUrl]?

Reply:
the find is necessary to get the first non-empty value.

Reply:
for example, if you pass `dev+proxy`, this query is equivalent to:

`API_TARGETS['proxy'] || API_TARGETS['dev'] || API_TARGETS['prod']`

Reply:
Ok, can you use a reducer instead of map + reverse + find ? + Add a comment why (_ex dev+proxy, proxy can be empty etc._ )

Reply:
if you consider `find` to be an specialized form of `reduce`, I'm already doing that. :P

Fine. I'll try to rewrite with reduce. I'm not sure if it will be more readable, though.

Reply:
yeah, the most important thing here is the comment :grin:

Reply:
ok. I stand corrected. it got simpler with reduce.